### DISCUSSION TITLE
Concluindo Curso Dominios Ricos - Qual próximo passo?

### DISCUSSION THREAD

**Discussion by hygor-melo** [balta-io/forum#397] [Tira Dúvidas] [answered]
Boa noite! estou concluindo o curso de modelando domínios ricos, gostei bastante do padrão que foi utilizado e queria saber se tem algum outro curso que aprofunde a criação de api usando aquela estrutura. Agradeço

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @hygor-melo, como vai? Tem sim, o Refatorando para testes de unidade, Aplicando OOP em cenários reais e Segurança em APIs ASP.NET.
